src/Traits/QueryComponentTraits/QueryComponentFilterTrait.php

```diff
@@ -93,10 +109,8 @@ protected function buildExcludes(): void
     {
         $filters = $this->query->getExclude();
         foreach ($filters as $field => $value) {
-            $value = is_array($value) ? $value : [$value];
-            $criteria = Criteria::where($field)
-                ->in($value)
-                ->not();
+            $criteria = $this->convertFilterToCriteria($field, $value);
```

Comment:
Can these two be combined in to a single
`$criteria = $this->buildCriteriaFilter()->not()`?